gradio/components.py

```diff
@@ -4068,11 +4084,11 @@ def postprocess(
             base_img_path = base_img
             base_img = np.array(_Image.open(base_img))
         elif isinstance(base_img, np.ndarray):
-            base_file = processing_utils.save_array_to_file(base_img)
```

Comment:
We can remove this method `processing_utils.save_array_to_file()` from `processing_utils.py`

Reply:
Sounds good - its internal but didn't want to delete anything ahead of 4.0